### ISSUE TITLE
"label and name from content mismatch" needs to also fail split up labels

### ISSUE THREAD

**Issue by ShadowBB** [act-rules/act-rules.github.io#451]
The "label content name mismatch" rule is now only applicable when there is an aria-label or aria-labelledby attribute. But this means it won't fail when situations where the visible text content and accessible name mismatch when there is no aria involved.

In addition I think it is best to give explicit examples where visible labels are split up by invisible parts of the accessible name. For example:

`<a href="home.html"> link <span style="sr-only">that opens a new screen</span> to the homepage</a>`

In contrast we could limit the scope of the rule to just cases with aria. But if we choose to do so we should definitely only look at non-empty aria-labels. (and we should also make it explicit that if the visible name is contained in the accessible name but with other content in between, it should still fail).

https://auto-wcag.github.io/auto-wcag/rules/SC2-5-3-label-content-name-mismatch.html

**Comment by annethyme** [comment]
... and have something like this as applicable + passed: 

<a href="home.html">Read more<span style="sr-only"> about our pricing</span></a>

**Comment by annethyme** [comment]
Working on it in this pull request: https://github.com/auto-wcag/auto-wcag/pull/447

**Comment by Jym77** [comment]
Reopening issue since (inactive) PR has been closed.

**Comment by WilcoFiers** [comment]
Related to #1619
